### PR TITLE
v0.5.0

### PR DESCRIPTION
**Pull request by tomschenkjr**
API
  * API documentation specifies an `error` field which provides any error messages from the request. (#17, #14)
  * Error codes follow HTTP status code conventions

Application Form
  * Added application page that allows list of eligible hospitals to select and apply #2
  * Added required fields needed to apply (from both the clinical and technical point-of-contact)
  * Provided a terms of service (TOS) required to use the service #1 
  * Within the TOS, noted this is for children between 0 and 1 years old #10 
    * Likewise, have added an error code if the patient is over 1 years-old.
 
Clinical Content
  * Added FAQ section in both Spanish and English #16
  * Provided PDF download of both Spanish and English FAQ
  * Added a downloadable map of where elevated blood-lead levels exist throughout Chicago. #16 

_After reviews have approved this request, I will only merge the request when we've pushed v0.5.0 into production._

### CODE REVIEW

**Comment by tomschenkjr** [comment]
@RaedMan - Please review the [terms of service](https://github.com/Chicago/lead-safe-api-docs/blob/ebd0f539efbe1add06d4d5c8b15b35d177f82f8b/docs/tos.md) again for any issues.

There is one obvious sentence that needs clarifying:

> The risk score, defined as (XXXXXXXXXXX) will be sent to health centers through electronic health records (EHR) and physicians will have the opportunity[...]

however, we will fix that when we release v0.6.0 of the API.

**Comment by geneorama** [comment]
I noticed a broken link in the notes of the main page, the https is missing the "h"

```
[^1]: Potash _et al._ (2015) [Predictive Modeling for Public Health](ttps://dssg.uchicago.edu/wp-content/uploads/2016/01/p2039-potash.pdf). _Proceedings of the 21th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining_ pp. 2039-2047.
```

should be 
```
[^1]: Potash _et al._ (2015) [Predictive Modeling for Public Health](https://dssg.uchicago.edu/wp-content/uploads/2016/01/p2039-potash.pdf). _Proceedings of the 21th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining_ pp. 2039-2047.
```

**Comment by geneorama** [comment]
Talked to @tomschenkjr and sounds like @RaedMan is working on the eligibility form, but for now the link is blank in the Eligibility section.  Do we want to put in note that it's under construction or take the link out for now? 

Also I didn't see an open issue for this or a note in the wiki.  This seems like a big task, I'm wondering if I missed it in the milestones.

**Review by geneorama — APPROVED** [review]
I made a few minor changes and committed them to dev.

**Review by RaedMan — COMMENTED** [review]
Not sure where to add this in TOS:  

"The use of the API precludes healthcare provider(s) and all staff, including third-party contractors, from the healthcare facility from transferring data or publishing any results derived from its activities hereunder and will not disclose any results or other material based upon data provided by the City without written permission from the City."

This may need some editing.

File: docs/tos.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+In these TOS, “You” or “Your” refers to any individual or entity that uses or seeks to use the API Services and Data.
+
+## Current Practice
+
+Pediatricians are directed to test every Chicago child’s blood for lead and report high levels to the Chicago Department of Public Health (CDPH). According to the Centers for Disease Control and Prevention (CDC), the reference level of 5 micrograms/dL should be used to identify children with elevated blood lead levels and is based on U.S. children between the ages of 1 - 5 years old based on the 97.5th percentile of the National Health and Nutrition Examination Survey of blood lead distribution in children.
+
+Though lead is no longer used in paint, plumbing, gasoline and a myriad of other products in the United States, Chicago’s older housing stock creates a persistent problem as homes built prior to 1978 that have not been properly maintained may still have older paint that can result in exposure to lead in children. This is especially true in some communities on the south and west sides of Chicago.
```
Comment (geneorama) [inline docs/tos.md:13]: This is good information, but should it be part of the TOS?

File: docs/tos.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+## Background
+
+To ensure success in the fight against lead poisoning, this project is part of CDPH's overall plan to provide concrete steps that are being taken by the department and its partners to further reduce and eventually eliminate lead poisoning in Chicago’s children.
+
+This lead safe predictive model helps identify infants at risk of being lead poisoned in homes with lead paint. The model provides an opportunity to prevent lead paint exposure through high priority proactive home lead inspections and blood testing at an earlier age. The predictive model combines data from multiple sectors including public health, census, buildings and the county assessor’s office to create real-time interfaces that identify where at-risk children live.
```
Comment (geneorama) [inline docs/tos.md:19]: I'm not sure if we should have "background" in the terms of service, but wherever this content lands I propose that we use the terminology "infants and fetuses" here to make subsequent language easier. In the next paragraph we refer to "pregnant patients", which is confusing and perhaps technically incorrect.

File: docs/tos.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+The risk score, defined as (XXXXXXXXXXX) will be sent to health centers through electronic health records (EHR) and physicians will have the opportunity to be alerted to either order a blood lead test and/or request to visually inspect the homes of either their pregnant patients and their at-risk children for lead paint hazards.
```
Comment (geneorama) [inline docs/tos.md:21]: The [risk score](https://github.com/Chicago/lead-safe-api-docs/issues/21) is part of the v0.6 milestone, but how do we want to describe the risk score here?  

Also, I don't understand why we're saying that "physicians will have the opportunity to be alerted to either...". Will they be opting into the alerts?

In any event, I think the risk score documentation should be separate from the description of alerts, and I'm not sure that the risk score documentation belongs in the TOS.

File: docs/tos.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Access to the XXXXXX Environment is subject to Your compliance with these terms as well as development of a privacy policy, including clear and conspicuous disclosure to users of any practice whereby You or a third party collect or use personally identifiable information from users. In addition, You agree to fully comply with all applicable local, state and federal laws, rules and regulations relating to the collection of personally identifiable information, including personal health information and their associated privacy and security laws. Any violation of these requirements or failure to comply with Your privacy policy constitutes a violation of these TOS and is grounds for immediate termination of all access privileges to the API services and Data.
```
Comment (geneorama) [inline docs/tos.md:37]: I'm not sure what the XXXXXX Environment means, so I'm not sure how to propose better terminology.

**Review by geneorama — CHANGES_REQUESTED** [review]

File: docs/tos.md (hunk 1 above)
Comment (tomschenkjr) [reply docs/tos.md:13]: Since it was provided by CDPH, let's keep it unless they prefer to have different language.

File: docs/tos.md (hunk 2 above)
Comment (tomschenkjr) [reply docs/tos.md:19]: Likewise, to the above, we'll keep it unless CDPH wants something different.

File: docs/tos.md (hunk 3 above)
Comment (tomschenkjr) [reply docs/tos.md:21]: Yes, they will be provided information/alert/scores so the physician can order blood tests/visual inspections.

File: docs/tos.md (hunk 4 above)
Comment (tomschenkjr) [reply docs/tos.md:37]: Placeholder for the name of the application. Will be "Lead Safe". Will make it in the next release.

**Comment by tomschenkjr** [comment]
@RaedMan - I've included the language under the "[City Policies and Requirements Regarding Use of API Services](https://github.com/Chicago/lead-safe-api-docs/blob/09b0acb835db71d49f8f881dac6c5019d5a1993e/docs/tos.md#city-policies-and-requirements-regarding-use-of-api-services)".

Let me know your thoughts.

**Review by geneorama — APPROVED** [review]

**Comment by geneorama** [comment]
@tomschenkjr thanks for the clarifications about what's going into .5 vs .6.  This is good for .5.

**Comment by tomschenkjr** [comment]
v0.5.0 has been deployed to the production servers! I am merging this code.

I will be sending an email to health partners so they can conduct some testing.

@RaedMan - the new web-based documentation will be updated as well. You can show the Lead Safe documentation to team members, which they can view in browsers: https://chicago.github.io/lead-safe-api-docs/

**Comment by geneorama** [comment]
So far all the tests are passing (including expected errors), except for QA because there's a problem with the actual server.  George has opened a ticket with EKI on that.

**Pull request merged**